From 64d211317e124ca33b2c66c79ae5f9594f1cb8ee Mon Sep 17 00:00:00 2001
From: Liming Wang <liming.wang@windriver.com>
Date: Tue, 25 Mar 2014 15:19:53 +0800
Subject: [PATCH 097/628] watchdog: xilinxps: Migrate to clock controller

git://github.com/Xilinx/linux-xlnx.git xilinx-v14.7
commit 28581ed9 arm: zynq: Migrate to clock controller

cherry picked the wdt driver modification.

Signed-off-by: Liming Wang <liming.wang@windriver.com>
---
 drivers/watchdog/xilinx_wdtps.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/watchdog/xilinx_wdtps.c b/drivers/watchdog/xilinx_wdtps.c
index 4853a57..25b6ea4 100644
--- a/drivers/watchdog/xilinx_wdtps.c
+++ b/drivers/watchdog/xilinx_wdtps.c
@@ -349,7 +349,7 @@ static int xwdtps_probe(struct platform_device *pdev)
 	watchdog_set_nowayout(&xwdtps_device, nowayout);
 	watchdog_set_drvdata(&xwdtps_device, &wdt);
 
-	wdt->clk = clk_get_sys("CPU_1X_CLK", NULL);
+	wdt->clk = clk_get(&pdev->dev, NULL);
 	if (IS_ERR(wdt->clk)) {
 		dev_err(&pdev->dev, "input clock not found\n");
 		res = PTR_ERR(wdt->clk);
-- 
1.7.5.4

